logserver.go

```diff
@@ -236,7 +238,13 @@ func addToArchive(targz *tar.Writer, dfilename string, filename string) error {
 }
 
 func serveGzippedFile(w http.ResponseWriter, r *http.Request, path string, size int64) {
-	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	cType := "text/plain; charset=utf-8"
+	if strings.HasSuffix(path, ".gz") {
+		cType = extensionToMimeType(path[:len(path)-len(".gz")])
```

Comment:
why are we doing this? do we expect to serve gzipped pngs? or is this mostly about the json case? Add a comment please.

Reply:
Commented